api/generated/crds/csi-baremetal.dell.com_availablecapacities.yaml

```diff
@@ -0,0 +1,78 @@
+
```

Comment:
What is the reason to keep these crds in 2 repos at the same time?

Reply:
Discussed on the meeting. I will create issue for it